.github/workflows/draft_release.yml

```diff
@@ -1,5 +1,7 @@
 name: Draft Release
 on:
+  schedule:
+    - cron: '0 10 * * 1'
```

Comment:
maybe adding a comment is a good idea here. (I have no clue what 0 10 * * 1 corresponds to without looking it up)

Reply:
Done 👍🏾